tests/test_elements.py

```diff
@@ -123,6 +123,7 @@ def test_Blocks_inner(self):
             .buffer(20)
             .exterior
         )
+        streets = streets.set_crs(self.df_tessellation.crs)
```

Comment:
The CRS loss is caused by an incorrect indexing we do. The line 120 should be `streets.loc[35, 'geometry'] = (`, not `streets.loc[35] = (`.

Reply:
And the incorrect indexing we do is not on purpose for the test, correct?

Reply:
No, we are just adding an line geometry fully enclosed within another one to check that the inner rings are correctly resolved.